save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/OrganizationController.kt

```diff
@@ -239,7 +239,6 @@ internal class OrganizationController(
         Parameter(name = "isAbleToToBulkUpload", `in` = ParameterIn.QUERY, description = "new flag for bulk upload files ability", required = true),
     )
     @ApiResponse(responseCode = "200", description = "Successfully changed ability to bulk upload files.")
-    @ApiResponse(responseCode = "403", description = "Could not change ability to bulk upload files due to lack of permission.")
```

Comment:
I guess `403` is returned if user is not `SUPER_ADMIN`

Reply:
Returned it